components/BaseComponents.js

```diff
@@ -55,6 +62,14 @@ export const ButtonLabel = styled.Text`
   color: ${props => props.color || Colors.lightest};
 `;
 
+export const BigTitleLabel = styled(BigTitle)`
+  line-height: 54px;
+  display: flex;
+  align-items: center;
```

Comment:
These don't do anything since this is a child. Also, since you already have text-align, that's all you need to make it center aligned.

Reply:
Ok now I see how this was probably taken from `ButtonLabel` but that doesn't need `display: flex` or `align-items` either...we can remove both

Reply:
👍